.nuke

```diff
@@ -1 +0,0 @@
-Silk.NET.sln
```

Comment:
I don't know what this does just putting this message to give you a heads up that this happened

Reply:
Yep, NUKE upgrade did this.